tests/integration/targets/ec2_instance_hibernation_options/roles/ec2_instance_hibernation_options/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# defaults file for ec2_instance
+ec2_instance_owner: 'integration-run-{{ resource_prefix }}'
+ec2_instance_type: 't3.micro'
+ec2_instance_tag_TestId: '{{ resource_prefix }}-{{ inventory_hostname }}'
+
+vpc_name: '{{ resource_prefix }}-vpc'
+vpc_seed: '{{ resource_prefix }}'
+vpc_cidr: '10.{{ 256 | random(seed=vpc_seed) }}.0.0/16'
+
+subnet_a_az: '{{ ec2_availability_zone_names[0] }}'
+subnet_a_cidr: '10.{{ 256 | random(seed=vpc_seed) }}.32.0/24'
+subnet_a_startswith: '10.{{ 256 | random(seed=vpc_seed) }}.32.'
+subnet_b_az: '{{ ec2_availability_zone_names[1] }}'
+subnet_b_cidr: '10.{{ 256 | random(seed=vpc_seed) }}.33.0/24'
+subnet_b_startswith: '10.{{ 256 | random(seed=vpc_seed) }}.33.'
+
+first_iam_role: "ansible-test-sts-{{ resource_prefix | hash('md5') }}-test-policy"
```

Comment:
I guess we can use `tiny_prefix` instead.

Reply:
@alinabuzachis Can you please let me know where to use tiny_prefix? I used the file from ec2_instance. I changed nothing.

Reply:
Something like here I mean https://github.com/ansible-collections/amazon.aws/blob/main/tests/integration/targets/ec2_instance_iam_instance_role/defaults/main.yml
to replace this `{{ resource_prefix | hash('md5') }}`